packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js

```diff
@@ -255,7 +255,8 @@ const FlowStageRecipes = {
       orderFeedUpdateParams: {
         ...defaultFlowStageParams,
         prerequisite: p,
-        testName: 'Order Feed Update (after Simulate Seller Approval)',
+        testName: 'OrderProposal Feed Update (after Simulate Seller Approval)',
+        orderType: 'order-proposals',
```

Comment:
Should this be renamed `orderFeedType`? (As orderType would imply `Order` or `OrderProposal` standard OA term values)

Reply:
Good idea 👍

Reply:
This is done